models/user.js

```diff
@@ -0,0 +1,74 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const crypto = require('crypto');
+
+require('dotenv').config();
+
+const userSchema = new Schema({
+  username: 
+  {
+    type: String,
+    required: true,
+    minlength: 5,
+    unique: true
+  },
+  password:
+  {
+    type: String,
+    minlength: 8,
+    required: true
+  },
+  isAdmin: 
+  {
+    type: Boolean,
+    default: false
+  },
+  email:
+  {
+    type: String,
```

Comment:
```suggestion
    type: String,
    required: true,
```

Email should be required for newsletters, cal invites etc.